### PR TITLE
modify etc configs for production usage

### PR DESCRIPTION
**Pull request by whitewum**
### Checklist

- [x] I've run the tests to see all new and existing tests pass
- [x] If this Pull Request resolves an issue, I linked to the issue in the text above
- [x] I've informed the technical writer about the documentation change if necessary

### CODE REVIEW

File: etc/nebula-metad.conf.default [hunk 1]
```diff
@@ -34,6 +34,6 @@
 
 ########## Misc #########
 # The default number of parts when a space is created
---default_parts_num=100
+# --default_parts_num=100
```
Comment (critical27) [inline etc/nebula-metad.conf.default:37]: How about just remove them from conf.

File: etc/nebula-storaged.conf.default [hunk 2]
```diff
@@ -27,25 +27,36 @@
...
 # The default block cache size used in BlockBasedTable.
 # The unit is MB.
---rocksdb_block_cache=4
+--rocksdb_block_cache=1
 # The type of storage engine, `rocksdb', `memory', etc.
 --engine_type=rocksdb
 
 ############## rocksdb Options ##############
 --rocksdb_disable_wal=true
+--wal_ttl=3600
```
Comment (critical27) [inline etc/nebula-storaged.conf.default:51]: How about move wal_ttl to storage?

File: etc/nebula-metad.conf.default (hunk 1 above)
Comment (whitewum) [reply etc/nebula-metad.conf.default]: done

File: etc/nebula-storaged.conf.default (hunk 2 above)
Comment (whitewum) [reply etc/nebula-storaged.conf.default]: refact a bit

File: etc/nebula-metad.conf.production [hunk 3]
```diff
@@ -31,9 +31,3 @@
 ########## storage ##########
 # Root data path, here should be only single path for metad
 --data_path=data/meta
-
```
Comment (critical27) [inline etc/nebula-metad.conf.production:34]: Sorry for I miss the `heartbeat_interval_secs` in `nebula-storaged.conf.production`, please also add heartbeat_interval_secs=10 here, it need to be matched in two configuration. 

That's why the `show hosts` would show different things over different times.

**Pull request merged**